### PR TITLE
UWP compat doc

### PR DESCRIPTION
**Pull request by krwq**
Initial version of doc with APIs supported on .NET Core but not on UWP

**For preview only - see wiki page**

Wiki: https://github.com/dotnet/corefx/wiki/UWP-Compat

### CODE REVIEW

File: Documentation/project-docs/uwp-compat.md [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+# System.Runtime*
+Note: This data was collected from comments on UWP specific disabled tests
+- [uapaot] Assembly.Load(byte[])")] not supported
+- [uap, not uapaot] When getting resources from PRI file the casing doesn't matter, if it is there it will always find and return the resource
+- [uapaot] Non-zero lower-bounded arrays not supported 
+- [uapaot] Exception.TargetSite always returns null
+
+# System.Private.CoreLib
+
+```
```
Comment (danmosemsft) [inline Documentation/project-docs/uwp-compat.md]: Can you help digest/summarize this list? Eg., for all the serialization ones, you can say that all serialization constructors and ISerializable implementations will throw PNSE for types that aren't marked [Serializable] -- those that are are all listed here: https://docs.microsoft.com/en-us/dotnet/standard/serialization/binary-serialization#binary-serialization-in-net-core

That would make it easier to diagest/rationalize than an API list.

Such comments can be gathered from the OneNote in many cases...

File: Documentation/project-docs/uwp-compat.md [hunk 2]
```diff
@@ -0,0 +1,489 @@
+# Behavior change between .NET Core and UWP
+
+This document contains list of not supported APIs on UWP but supported on .NET Core
+
+## Notes
+- This document is for preview only and was semi-generated, information may be incomplete or may contain errors
```
Comment (jkotas) [inline Documentation/project-docs/uwp-compat.md]: Would it be better for the doc to live on the wiki, next to https://github.com/dotnet/corefx/wiki/ApiCompat ?

Comment (krwq) [inline Documentation/project-docs/uwp-compat.md]: I can move it over there when this review is finished although I find regular files easier to work with as you can i.e. send a PR for review

Comment (krwq) [inline Documentation/project-docs/uwp-compat.md]: moved it to wiki https://github.com/dotnet/corefx/wiki/UWP-Compat

File: Documentation/project-docs/uwp-compat.md (hunk 1 above)
Comment (krwq) [inline Documentation/project-docs/uwp-compat.md]: done

**Comment by krwq** [comment]
Closing, please directly edit wiki page

**Pull request closed without merging**